app/src/main/java/com/sopetit/softie/ui/dailyroutine/dailyroutineadd/DailyRoutineAddCardPagerAdapter.kt

```diff
@@ -0,0 +1,39 @@
+package com.sopetit.softie.ui.dailyroutine.dailyroutineadd
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import coil.load
+import com.sopetit.softie.databinding.ItemDailyRoutineAddCardBinding
+import com.sopetit.softie.domain.entity.DailyCard
+import com.sopetit.softie.util.ItemDiffCallback
+import timber.log.Timber
+
+class DailyRoutineAddCardPagerAdapter :
+    ListAdapter<DailyCard, DailyRoutineAddCardPagerAdapter.DailyPagerViewHolder>(
+        ItemDiffCallback<DailyCard>(
+            onItemsTheSame = { oldItem, newItem -> oldItem == newItem },
```

Comment:
이 부분에 item이 고유한 값으로 지닌 값으로 비교해주세요~
예시로,

```suggestion
            onItemsTheSame = { oldItem, newItem -> oldItem.themeId == newItem.themeId },
```